src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/ValidationServiceFacadeImpl.java

```diff
@@ -84,11 +99,11 @@ public void auditJsr(long jobId) {
                 exceptionCaseWorker.setFieldInError(constraintViolation.getPropertyPath().toString());
                 exceptionCaseWorker.setErrorDescription(constraintViolation.getMessage());
                 exceptionCaseWorker.setExcelRowId(String.valueOf(constraintViolation.getRootBean().getRowId()));
-                exceptionCaseWorker.setUpdatedTimeStamp(LocalDateTime.now());
+                exceptionCaseWorker.setUpdatedTimeStamp(LocalDateTime.now(ZoneOffset.UTC));
                 exceptionCaseWorker.setKeyField(getKeyFieldValue(field.get(), constraintViolation.getRootBean()));
```

Comment:
When persisting an OffsetDateTime entity attribute, Hibernate does not store the offset separately, hence the local offset is used. For this reason, the OffsetDateTime is not very useful, and it can be replaced by a LocalDateTime instead.(https://vladmihalcea.com/date-timestamp-jpa-hibernate/) Using Zonset here hence time stored in UTC